internal/stack/_static/local-services-docker-compose.yml.tmpl

```
@@ -76,7 +76,7 @@ services:
     - "host.docker.internal:host-gateway"
 
   elastic-agent_is_ready:
-    image: tianon/true:multiarch
+    image: "${ISREADY_IMAGE_REF}"
```

This image should be using also fact as the previous service:

```suggestion
    image: "{{ fact "isready_image" }}"
```